src/clj/rems/db/csv.clj

```diff
@@ -0,0 +1,115 @@
+(ns rems.db.csv
+  "Utilities for exporting database contents as CSV"
+  (:require [clj-time.core :as time]
+            [clojure.string :as str]
+            [rems.config :refer [env]]
+            [rems.db.user-settings :as user-settings]
+            [rems.text :as text]))
+
+(defn print-to-csv [& {:keys [column-names rows
+                              enclose-strings-in-quotes separator]
+                       :or {separator (:csv-separator env)}}]
+  (let [escape-quotes #(str/replace % "\"" "\\\"")
```

Comment:
I'm not sure this gets all quotes right. What if the value already has an escaped quote `\"`? Did you try importing values quoted like this in excel? Not a huge thing, csv doesn't really have a spec so as long as we work on most data it's fine.

Reply:
Thanks for the comment, I did not test importing the output to Excel (or rather, LibreOffice Calc) before, but I did now. Some observations:

- An escaped quote `\"` in the value seems to work fine when importing. It looks like Calc only treats the escape character specially if it directly precedes the text delimiter character (i.e., a quote).

- However, if the value ends with a quote `"`, Calc doesn't interpret it correctly, but instead messes up the subsequent columns. I think it is a problem in Calc because `"string\""` looks to me like it's correctly escaped and interpretable. The problem might be caused by there being an alternative way to escape `"` as `""`, which seems to be implemented at least in R's `write.csv` function. But I don't think this is actually our problem.

- At least Python and R seem to use a similar method of escaping only the text delimiter character as I did here, so this seems to be consistent with how it's done elsewhere.